apps/backend/src/utils/country.utils.ts

```diff
@@ -0,0 +1,18 @@
+import { CountryEnum } from "src/core/port/domain/enums/country.enum";
+
+export function validateCountries(countryDeparture: string, countryArrival: string): string {
```

Comment:
**suggestion:** Try to avoid "utils" or "common" stuff. Even if it is natural to try to generalize stuff, or to "re-use", it is generaly a bad idea as it will be much harder to understand what is happening in the flow of the app, throw appripriate errors, and most of all losing the intent of the user that made the api call.

Instead of this being a function lost in utils, it should be part of your domain business layer, or at least you application layer, for example in RouteService if you don't have a dedicated Country aggregate or service